CHANGELOG.md

```diff
@@ -5,14 +5,38 @@ versioning](http://semver.org).
 ## [Unreleased] - [unreleased]
 ### Added
 - Traceback is logged when a service module can't be imported
+- Improvements to release cycle documentation
+- Configuration parameter for ``$.controlPanel`` to set height of open panel
+  explicitly
+- Documentation has an explicit "hello world" example in the Quick Start section
+- Quiet option reduces verbosity
+- "Watch" plugin controls whether services and dependent modules are
+  automatically reloaded when they change
 
 ### Changed
+- Documentation introduction is more focused; tutorials are more
+  front-and-center
+- Updated bundled version of GeoJS
+- Support Travis containers for better automated testing
+- Verbose option can be specified multiple times to increase application
+  verbosity
+- ``--config`` option can now accept a raw JSON string in place of a YAML
+  filename
```

Comment:
I might word this as "can now accept either a YAML filename or a raw JSON string" so that it is obvious it can do either.